---
title: "Research"
output: html_document
---

Like most biologists I am fascinated by the natural variation in nature. My research is generally focused on translating this variation to understand how it has developed and how it persists. A lot of my PhD research is focused on agnostically capturing the large scale patterns in phylogenetic data. I see this as a first step towards better understanding the  identifying the sources of variation, whether there are biological abnormalities, or systematic bias introduced by inadequate models. 


## Identifying and visualizing structure in phylogenetic tree sets


<a href="https://treescaper.github.io/TreeScaperNetworks.html">
<img src="images/Distance74edge61start.jpg" style="width:50%; margin-right: 20px" align="right">
</a>

Variation in inferred gene trees is one of the most striking and consistent observations in phylogenomics. Unraveling the nature and source of this variation is crucial to understanding fundamental aspects of evolution such as molecular evolution across genomes, rates of speciation, hybridization, and patterns of phenotypic evolution. I used simulations to investigate the performance of recently developed network-based community detection approaches , implemented in [TreeScaper](https://github.com/whuang08/TreeScaper/releases), for the identification of such structure. Implementing network methods for identifying structure and clustering information in phylogenetic datasets frees researchers from the loss of information inherent to dimensionality reduction methods. 




							